database/clickhouse/tests/v2/clickhouse_test.go

```diff
@@ -0,0 +1,232 @@
+package clickhouse_test
+
+import (
+	"context"
+	"database/sql"
+	sqldriver "database/sql/driver"
+	"fmt"
+	"log"
+	"net/url"
+	"testing"
+
+	_ "github.com/ClickHouse/clickhouse-go/v2"
```

Comment:
Are the tests split to support both v1 and v2? What's involved in supporting both versions and why would we not update to v2 and drop support for v2? I'm not a fan of duplicate code.

Reply:
some people use golang-migrate as part of their application. Go driver V1 and V2 are not compatible and it will be impossible to import them in the same application

Reply:
@kshvakov I like the idea to support both for a limited time, on the other hand, for the CLI, we can only support one, so we have to decide if that should be v1 or v2.